### PR TITLE
O3-1253: Highlight abnormal vitals

### PR DESCRIPTION
**Pull request by denniskigen**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary

Abnormal values for vital signs should be flagged in both the vitals header and the vitals widget.

This PR highlights abnormal vitals in the vitals widget as described in the ticket https://issues.openmrs.org/browse/O3-1253.

## Screenshots

Screenshot 2022-05-13 at 17 46 58

## Related Issue

https://issues.openmrs.org/browse/O3-1253

### CODE REVIEW

File: packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.utils.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import isNumber from 'lodash/isNumber';
 import { ConceptMetadata } from '@openmrs/esm-patient-common-lib';
 
 export function calculateBMI(weight: number, height: number): number {
+  if (!weight && !height) return;
```
Comment (ZacButko) [inline packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.utils.ts:6]: Should this be `||` instead? 

Also should this function have a default return value?

File: packages/esm-patient-vitals-app/src/vitals/vitals.resource.tsx [hunk 2]
```diff
@@ -168,3 +208,82 @@ export function editPatientVitals(
     },
   });
 }
+
+export function exists(...args: any[]): boolean {
```
Comment (ZacButko) [inline packages/esm-patient-vitals-app/src/vitals/vitals.resource.tsx:212]: Should this function be called `allExist`?  The difference of 'some' exist vs 'every' exist.

**Review by ZacButko — APPROVED** [review]
I'm curious about a few things but otherwise looks great.

File: packages/esm-patient-vitals-app/src/vitals/vitals.resource.tsx (hunk 2 above)
Comment (denniskigen) [reply packages/esm-patient-vitals-app/src/vitals/vitals.resource.tsx:212]: Turns out this function is not strictly necessary.

File: packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.utils.ts (hunk 1 above)
Comment (denniskigen) [reply packages/esm-patient-vitals-app/src/vitals/vitals-biometrics-form/vitals-biometrics-form.utils.ts:6]: Yeah, it should. I don't think it should have a default return value.

**Pull request merged**